panc/src/test/java/org/quattor/pan/PanTest.java

```diff
@@ -0,0 +1,100 @@
+package org.quattor.pan;
+
+import org.quattor.pan.exceptions.SyntaxException;
+import java.util.*;
+
+/**
+ * Created by iliclaey.
+ *
+ * When building the panc-project a panc-10.3-SNAPSHOT-fat-jar-with-tests.jar file is created.
+ * This jar can be used to execute pan test files and should be used as follows:
+ * `java -Dpanc.testdir="path" -Dpanc.tmpdir="path" -jar path/to/panc-10.3-SNAPSHOT-fat-jar-with-tests.jar`.
+ * The panc.testdir and panc.tmpdir properties need to be set, otherwise an error will be thrown.
+ *
+ * To be able to execute your tests, they need to conform to a specific directory structure.
+ * The directory you pass to the panc.testdir property needs to contain the _Functionality_,  _Dependency_ or
+ * _RootElement_ directories, depending on what type of tests you want to run. Each of these directories need to be
+ * divided in subdirectories, which contain the test files (or more subdirectories).
+ *
+ * panc.testdir
+ * |-- Dependency
+ * |-- Functionality
+ * |   |-- subdir1
+ * |   |   |-- file1.pan
+ * |   |   |-- file2.pan
+ * |   |-- subdir2
+ * |   |   |-- file1.pan
+ * |   |   |-- file2.pan
+ * |-- RootElement
+ *
+ * Dependency tests
+ * ================
+ * A simple example:
+ *
+ * # dep: simple
+ * object template simple;
+ *
+ * Functionality tests
```

Comment:
describe this one before the `Dependecy tests`, as it is the most common one (or even only one)